templates/login.liquid

```diff
@@ -0,0 +1,174 @@
+    <style>
+      :root {
+        --forum-instructions-width: 300px;
+        --forum-instructions-margin: 20px;
+      }
+      .ea-layout {
+        display: flex;
+        width: auto;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+      }
+      @media (min-width: 480px) {
+        .ea-layout {
+          width: 100%;
+        }
+      }
+      @media (min-width: 1070px) {
+        .ea-layout {
+          padding: 80px 0;
+        }
+        .ea-layout-forum {
+          flex-direction: row;
+          align-items: flex-start;
+        }
+        .ea-layout-contact {
+          flex-direction: column;
+          align-items: center;
+        }
+      }
+      .forum-instructions {
+        width: 100%;
+        border-radius: 5px;
+        background-color: #dae0e2;
+        padding: 20px;
+      }
+      @media (min-width: 480px) {
+        .forum-instructions {
+          width: 400px;
+        }
+      }
+      @media (min-width: 1070px) {
+        .forum-instructions {
+          margin-right: var(--forum-instructions-margin);
+          width: var(--forum-instructions-width);
+        }
+      }
+      .forum-instructions p {
+        margin-bottom: 1em;
+        line-height: 1.5;
+      }
+      .forum-instructions-details-text p:last-child {
+        margin-bottom: 0;
+      }
+      .forum-instructions-hide-this {
+        margin-top: 1em;
+        margin-bottom: 0;
+      }
+      @media (min-width: 1070px) {
+        .forum-instructions-hide-this {
+          display: none;
+        }
+      }
+      .forum-instructions-show-more {
+        margin-top: 1em;
+        margin-bottom: 0;
+        display: none;
+      }
+      .widget-wrapper {
+        width: 100%;
+      }
+      @media (min-width: 480px) {
+        .widget-wrapper {
+          width: 400px;
+        }
+      }
+      @media (max-width: 480px) {
+        ._prompt-box-outer {
+          width: 100vw;
+        }
+      }
+      .forum-instructions-balancer {
+        visibility: hidden;
+      }
+      @media (min-width: 1070px) {
+        .forum-instructions-balancer {
+          width: calc(var(--forum-instructions-width) + var(--forum-instructions-margin));
+        }
+      }
+      .contact-us {
+        width: 100%;
+        height: calc(2em+5px);
+        background-color: #dae0e2;
+        padding: 10px;
+        border-radius: 5px;
+        text-align: center;
+        position: relative;
+        z-index: 5;
+      }
+      @media (min-width: 480px) {
+        .contact-us {
+          margin-top: 20px;
+          margin-bottom: 20px;
+          width: 400px;
+        }
+      }
+      @media (min-width: 1070px) {
+        .contact-us {
+          margin-bottom: 0;
+        }
+      }
+    </style>
+    <script>
+      function hideForumDetails() {
+        const details = document.getElementById("forum-instructions-details");
```

Comment:
You're targeting DOM elements by ID, and that's fine, but (given that you've duplicated the class name and the ID name) just to point out that you could reduce duplication by either:

- Making the CSS target the element ID, not the class (e.g. `#forum-instructions-details { /* styles */ }`
- Using `document.querySelector()` to get the DOM element. 

Probably the former is better (because it's a unique element you want to find, and IDs are unique, whereas classes are open to re-use, and then you either have to ensure that `document.querySelector()` is getting the first instance of the class in the DOM, or you use `document.querySelectorAll()` and then you have to iterate). Anyway, no changes necessary really, more of an FYI...